django/contrib/gis/static/gis/js/OLInit.js

```diff
@@ -0,0 +1,23 @@
+/*global MapWidget*/
+
+document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
I'd probably merge these both into OLMapWidget, then you won't need to export MapWidget onto window